apt/minimization/minimizer.py

```diff
@@ -66,7 +66,7 @@ def __init__(self, estimator: Union[BaseEstimator, Model] = None, target_accurac
                  encoder: Optional[Union[OrdinalEncoder, OneHotEncoder]] = None,
                  features_to_minimize: Optional[Union[np.ndarray, list]] = None,
                  train_only_features_to_minimize: Optional[bool] = True,
-                 is_regression: Optional[bool] = False):
+                 is_regression: Optional[bool] = False, accuracy_score: Callable = None):
```

Comment:
Please add this new parameter to the class docstring, stating clearly what are the expectations from this callable (inputs and outputs).